server/apps/immich/src/modules/immich-jwt/strategies/jwt.strategy.ts

```diff
@@ -13,8 +13,7 @@ export class JwtStrategy extends PassportStrategy(Strategy, 'jwt') {
   constructor(
     @InjectRepository(UserEntity)
     private usersRepository: Repository<UserEntity>,
-
-    private immichJwtService: ImmichJwtService,
+    immichJwtService: ImmichJwtService,
```

Comment:
just for my own knowledge, why do we need to remove `private` here?

Reply:
It's not strictly necessary, but adding private makes it available as a private property on the class (this.configService). However it's not used anywhere so vscode greys it out, like it's not used. Basically it doesn't need it since it's only referenced in the constructor, without `this.`

Reply:
https://stackoverflow.com/a/48571668/2833217